app/api/helpers/query.py

```diff
@@ -0,0 +1,41 @@
+from flask import request
+from flask_rest_jsonapi.exceptions import ObjectNotFound
+
+from app.api.helpers.db import safe_query
+from app.models.event import Event
+from app.api.helpers.permission_manager import has_access
+
+
+def event_query(self, query_, view_kwargs, event_id, event_identifier, permission):
```

Comment:
Thanks for pointing it out @poush , the function call length reduced because of that :smile: